cli-commands/shape/shape.js

```diff
@@ -30,32 +25,15 @@ const run = async (name) => {
         
     let repo = getRepo(name)
 
-    //if a user pass his own repo to be integrated with etherlime
     if(repo.includes(undefined)) {
```

Comment:
I  have never seen a check for includes undefined. It should be something else. Please find the best way to check if the name is undefined :)